internal/config/config.go

```diff
@@ -153,7 +168,7 @@ func Load(cfgFile string) (*Config, error) {
 	if err != nil {
 		return nil, fmt.Errorf("reading config file: %v", err)
 	}
-	c := &Config{}
+	c := NewDefault()
```

Comment:
Pay attention to this change in behavior - it uses the defaults if no value is specified